frontends/p4/fromv1.0/converters.cpp

```diff
@@ -416,6 +416,35 @@ ExternConverter *ExternConverter::get(cstring type) {
 
 ///////////////////////////////////////////////////////////////
 
+std::map<cstring, std::vector<PrimitiveConverter *>> *PrimitiveConverter::all;
+
+PrimitiveConverter::PrimitiveConverter(cstring name, int prio) : prim_name(name), priority(prio) {
+    static std::map<cstring, std::vector<PrimitiveConverter *>> converters;
```

Comment:
Because of the !@$&! undefined order of global static initialization in C++.  If the map was in one compilation unit and an instance static was in another, it *might* initialize the instance before the map, leading to undefined behavior.  By making the map itself a static local in the constructor, it is guarenteed to be initialized the first time the constructor is run.  I'll rename the var to be more descriptive.